src/core/index.ts

```diff
@@ -102,6 +146,13 @@ export interface Configuration {
     autoResubscribe?: boolean;
 }
 
+export type PromptReminder =
+    | {
+          promptUuid: string;
```

Comment:
From read on the code, is there redundancy here recording the UUID in the object? Nothing seems to use the prop as the key already holds UUID and we only care about the value.

Not too fussed if we think there could be a use for it in future, doubt it's much storage overhead.

Reply:
Stuck this back in after removing it the first time i used the uuid in the entry key - will chop